lib/facter/rhsm_manage_repo.rb

```diff
@@ -0,0 +1,15 @@
+Facter.add('rhsm_manage_repo') do
+  confine osfamily: 'RedHat'
+  setcode do
+    rhsm_result = Facter::Util::Resolution.exec('subscription-manager config --list  | grep manage_repos | awk \'{ print $3}\'')
```

Comment:
Small suggestion. If I'm grabbing the last field with awk, I generally use `$NF` instead of the field number. It makes it slightly more failsafe in case the output ever changes.

```suggestion
    rhsm_result = Facter::Util::Resolution.exec('subscription-manager config --list  | grep manage_repos | awk \'{ print $NF}\'')
```

Reply:
Good suggestion, added.